grpc/authz/authz.go

```diff
@@ -0,0 +1,96 @@
+package authz
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	jwt "github.com/cristalhq/jwt/v3"
+	"github.com/go-logr/logr"
```

Comment:
It doesn't appear `logr` is being used other than to define the `Log` attribute that doesn't appear to be called anywhere. Can we remove that dependency since it's not used?

Reply:
ah yes, sorry. leftover from developing and debugging.